website/source/layouts/docs.erb

```diff
@@ -268,8 +268,8 @@
             <a href="/docs/auth/approle.html">AppRole</a>
           </li>
 
-          <li<%= sidebar_current("docs-auth-aws-ec2") %>>
-            <a href="/docs/auth/aws-ec2.html">AWS EC2</a>
+          <li<%= sidebar_current("docs-auth-aws") %>>
+            <a href="/docs/auth/aws-ec2.html">AWS</a>
```

Comment:
As I stated in my very first comment, I just didn't know how you wanted to handle it. Sounds like you're saying I should just rename it and you'll add the higher level redirect? Done, but in a separate commit so it should be easy to roll back.